rules/utils/import-utils.js

```diff
@@ -1,41 +1,196 @@
+module.exports.getImportAccessPath = getImportAccessPath;
```

Comment:
Utility files should be outside of the `rules` directory.

Reply:
I hope to use it later in the detect-child-process rule as well. #104
So the `utils` directory is good for me.

Reply:
That's fine, but it shouldn't be in the `rules` directory. Please move it to the top level.